Cargo.toml

```diff
@@ -51,3 +52,8 @@ create-release = false
 [profile.dist]
 inherits = "release"
 lto = "thin"
+
+[profile.release.package.sql-experimental]
+debug = false
+strip = true
+opt-level = "z"
```

Comment:
`z` must have very good reasoning behind it. It has two issue:
1. I've seen multiple times that `z` actually produced slightly larger binaries than `s`
2. `z` is aggresive size optimization at all cost, including performance; we generally don't want to sacrifice performance for a few bytes, and that's what `z` usually offers. It pretty much only makes sense if you compile to your microcontroller with 4KiB of flash memory and your program didn't fit in there when compiled with `s`

Reply:
`z` disables vectorization and SIMD operations resulting in substantially slower code. This should not be the default build configuration.